Diagnose-PrinterBlockTimeline.ps1

```
@@ -0,0 +1,193 @@
+
+# --- 808 bursts (collapse into 10-minute buckets) ----------------------------
+$blocks = @()
+try {
+    $blocks = Get-WinEvent -FilterHashtable @{ LogName='Microsoft-Windows-PrintService/Admin'; Id=808; StartTime=$since } -ErrorAction Stop
+} catch { }
+$earliest808Overall = $null
+try {
+    $allBlocks = Get-WinEvent -FilterHashtable @{ LogName='Microsoft-Windows-PrintService/Admin'; Id=808 } -MaxEvents 1000 -ErrorAction Stop |
+        Sort-Object TimeCreated
+    if ($allBlocks) { $earliest808Overall = $allBlocks[0].TimeCreated }
+} catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)